app/src/main/java/manparvesh/ideatrackerplus/MainActivity.java

```diff
@@ -2017,6 +2038,11 @@ public void onCheckedChanged(IDrawerItem drawerItem, CompoundButton buttonView,
                     mTinyDB.putBoolean(getString(R.string.big_text_pref), false);
                     DatabaseHelper.notifyAllLists();
                 }
+                break;
+
+            case 7:
```

Comment:
`case 7:` Should be a constant value to make it easily understandable, similar to other items of switch case `ID_TOGGLE_DONE` and `ID_TOGGLE_BIG_TEXT`